src/Akeneo/Connectivity/Connection/front/src/error-management/components/ErrorList/ErrorMessageCell.tsx

```diff
@@ -0,0 +1,100 @@
+import styled from '../../../common/styled-with-theme';
+import {ConnectionErrorContent} from '../../model/ConnectionError';
+import {DocumentationList} from '../Documentation/DocumentationList';
+import {ErrorProductInformation} from './ErrorProductInformation';
+
+const messageWithColoredParameters = (template: string, parameters: {[param: string]: string}) => {
+    const messageComponents = Object.entries(parameters).reduce(
+        (messageComponents, [key, value]) => {
+            return messageComponents
+                .map((messageComponent, i) => {
+                    if (typeof messageComponent !== 'string') {
+                        return messageComponent;
+                    }
+
+                    const messageParts = messageComponent.split('{' + key + '}');
+
+                    if (1 === messageParts.length) {
+                        return messageParts;
+                    }
+
+                    messageParts.splice(1, 0, <ColoredParameters key={i}>{value}</ColoredParameters>);
+
+                    return messageParts;
+                })
+                .flat();
+        },
+        [template]
+    );
+    return messageComponents;
+};
+
+type Props = {
+    content: ConnectionErrorContent;
+};
+
+const ErrorMessageCell: FC<Props> = ({content}) => {
+    return (
+        <Container>
+            {content?.product && <ErrorProductInformation product={content.product} />}
+            {'domain_error' === content?.type &&
+            'string' === typeof content?.message_template &&
+            null !== content?.message_parameters ? (
+                <ErrorMessage>
+                    {messageWithColoredParameters(content?.message_template, content.message_parameters)}
+                </ErrorMessage>
+            ) : (
+                <table>
+                    <tbody>
+                        {Object.entries(content)
+                            .filter(
+                                ([key]) =>
+                                    'product' !== key &&
+                                    'documentation' !== key &&
+                                    'message_template' !== key &&
+                                    'message_parameters' !== key &&
+                                    'locale' !== key &&
+                                    'scope' !== key
+                            )
+                            .map(([key, value], i) => {
+                                return (
+                                    <ErrorLine key={i}>
+                                        <ErrorKey>{key}</ErrorKey>
+                                        <ErrorValue>: {JSON.stringify(value)}</ErrorValue>
+                                    </ErrorLine>
+                                );
+                            })}
+                    </tbody>
+                </table>
+            )}
+            {content.documentation !== undefined && <DocumentationList documentations={content.documentation} />}
```

Comment:
You can use the short notation in those cases
```suggestion
            {content.documentation && <DocumentationList documentations={content.documentation} />}
```